prboom2/src/m_menu.c

```diff
@@ -1985,8 +1985,7 @@ static void M_DrawSetting(const setup_menu_t* s)
 
     if (key) {
       M_GetKeyString(*key,0); // string to display
-      if (key == &key_up || key == &key_down || key == &key_speed ||
-         key == &key_fire || key == &key_strafe || key == &key_strafeleft || key == &key_straferight || key == &key_use)
+      if (s->m_mouse || s->m_joy)
```

Comment:
Incidentally, I removed this branch condition entirely in dsda-doom because it doesn't seem to serve any purpose https://github.com/kraflab/dsda-doom/commit/8c8dadb6247e0690d0a35d0d43db6b79f9c47efb